lib/docx/body-reader.js

```diff
@@ -312,8 +312,10 @@ function BodyReader(options) {
             
             var gridSpan = properties.firstOrEmpty("w:gridSpan").attributes["w:val"];
             var colSpan = gridSpan ? parseInt(gridSpan, 10) : 1;
+            var isTableHeader = properties.children && properties.children[0].attributes['w:firstColumn'] === '1';
             
             var cell = documents.TableCell(children, {colSpan: colSpan});
+            cell.isTableHeader = isTableHeader;
```

Comment:
I knew you wouldn't like that but the options object wasn't getting parsed into the `convertTableCell` function in the `document-to-html.js` file. The cell element _was_ getting parsed into the `convertTableCell` function though so that's the only way I was able to send the table header status data to the function.

I did try to look for a way to parse the options object into the function but I couldn't figure out how.